virt/virt-2-4-release-notes.adoc

```diff
@@ -4,7 +4,9 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-release-notes
 toc::[]
 
-== About {VirtProductName} {VirtVersion}
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} is now a fully supported and generally available feature of the {product-title} 4.5. Previously known as container-native virtualization, {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside with containers, and are managed as native Kubernetes objects.
```

Comment:
depending on product-title `the` looks superfluous - but I'm not a native speaker.

Reply:
Will check with peer reviewer on: "...generally available feature of the OpenShift Container Platform 2.4" - whether "the" is required or should be dropped.

Reply:
1. Fabian's right. Drop the 'the'
2. "..where they run alongside with containers.." <-- needs to drop the 'with', but perhaps instead it should be "..where they run alongside container workloads.." ?

Reply:
@fabiand 
Updated content:
Removed the article "the".
Removed the preposition "with".